database/buildSchema/37_reviewer_action_list.sql

```diff
@@ -23,6 +24,11 @@ CREATE FUNCTION review_list (stageid int, reviewerid int)
                 THEN 'RESTART_REVIEW'
             WHEN COUNT(*) FILTER (WHERE review_status_history.status = 'DRAFT') != 0
                 THEN 'CONTINUE_REVIEW'
+            WHEN COUNT(*) FILTER (WHERE review_assignment.status = 'ASSIGNED' 
+                                    AND review_assignment.is_final_decision = TRUE
+                                    AND review_assignment.is_last_stage = TRUE
+                                    AND review.id IS NULL) != 0
```

Comment:
Yes, good point. It would be nice to return that even before you have started - that was my intention even when building the diagram that is the case.